corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -194,3 +194,8 @@ def get_data_schema(request, domain, app_id=None, form_unique_id=None):
     if "pretty" in request.GET:
         kw["indent"] = 2
     return HttpResponse(json.dumps(data, **kw))
+
+
+@require_GET
```

Comment:
The session is handled by middleware, right? And all middleware should apply to all views unless we're doing something special to bypass the auth middleware for requests that do not require login. Do  you know if we're doing that?